pyproject.toml

```diff
@@ -1,28 +1,26 @@
 [tool.poetry]
 name = "indicodata-aiohttp_responses"
-version = "1.0.0"
+version = "1.0.1-rc0+dronev3.0"
 description = "proper aiohttp_responses lib"
 authors = ["indico <engineering@indicodata.ai>"]
 packages = [ {include = "aiohttp_responses"}]
 
 [tool.poetry.dependencies]
-python = ">=3.7,<4.0"
-setuptools = "^57.4.0"
-pytest = {version = "^6.2.4", extras = ["test"]}
```

Comment:
`test` wasn't an extra for pytest, and doesn't look like we used anything in the `testing` one